File: packages/sdk/android/src/main/java/com/primerioreactnative/extensions/klarna/KlarnaPaymentCategory.kt
package com.primerioreactnative.extensions.klarna

import com.primerioreactnative.datamodels.klarna.KlarnaPaymentCategoryRN
import io.primer.android.klarna.implementation.session.domain.models.KlarnaPaymentCategory

internal fun KlarnaPaymentCategory.toKlarnaPaymentCategoryRN() =
  KlarnaPaymentCategoryRN(
    identifier = identifier,
    name = name,
    descriptiveAssetUrl = descriptiveAssetUrl,
    standardAssetUrl = standardAssetUrl,
  )

File: packages/sdk/src/models/klarna/KlarnaPaymentCategory.ts
export type KlarnaPaymentCategory = IKlarnaPaymentCategory;

/**
 * Interface representing a Klarna payment category.
 */
interface IKlarnaPaymentCategory {
    /**
     * The identifier of the category.
     */
    identifier: string;
    /**
     * The name of the category.
     */
    name: string;
    /**
     * The descriptive asset url of the category.
     */
    descriptiveAssetUrl: string;
      /**
     * The standard asset url of the category.
     */
    standardAssetUrl: string;
}
